wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -166,6 +187,333 @@ static byte _tls2tsipdef(byte cipher)
     return def;
 }
 #endif
+
+
+
+
+#ifdef HAVE_AES_CBC
+
+static  int tsip_bench_aescbc_internal(
```

Comment:
See comment above about moving benchmark-specific code outside the library proper.  Maybe benchmark.c?

Reply:
Moved all the functions used only in Benchmark to benchmark.c.